homeassistant/components/econet/water_heater.py

```diff
@@ -239,9 +151,31 @@ def target_temperature(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return self.water_heater.min_set_point
+        return self.water_heater.set_point_limits[0]
 
     @property
     def max_temp(self):
         """Return the maximum temperature."""
-        return self.water_heater.max_set_point
+        return self.water_heater.set_point_limits[1]
+
+    @property
+    def should_poll(self) -> bool:
+        """Return True if entity has to be polled for state.
+
+        False if entity pushes its state to HA.
+        """
+        return self._poll
```

Comment:
Do we still need this to be a variable? This only affects the service calls.

Reply:
This forces a check of the heaters usage which is a REST call. It doesn't need to run every time, only if the unit's running status changes so I think this is still needed.